src/helpers/sceneless-project.ts

```diff
@@ -1776,6 +1795,37 @@ export const commands = (_project: ScenelessProject) => {
         sendState()
       })
     },
+
+    async addParticipantTrack(
+      trackId: string,
+      props: Partial<ParticipantProps> = {
+        isMuted: true,
+        isHidden: false,
+        volume: 0,
+      },
+      type: ParticipantType = 'camera',
+    ) {
+      return await commands.addParticipant(trackId, props, type)
```

Comment:
Down in line 1829, we have:
```typescript
 async addParticipant(
      participantId: string,
      props: Partial<ParticipantProps> = {},
      type: ParticipantType = 'camera',
)
```

But here, we call `addParticipant`, and pass `trackId` in as the `participantId`? This is confusing to me.

Reply:
Add a track to the stream in the same way you would add a participant.
The primary camera is assigned to the participant, while all other external cameras are assigned to ParticipantTrack.

We don't actually need a separate function, but it's there for the sake of differentiation for end customers. So they are aware of two functions, addParticipant and addParticipantTrack, both of which perform the same function.